pkg/nodediscovery/nodediscovery.go

```diff
@@ -525,6 +526,13 @@ func determineFirstInterfaceIndex(instanceType string) *int {
 		if defaults.IPAMPreAllocation > max {
 			idx = 0 // Include eth0
 		}
+	} else {
+		log.WithFields(logrus.Fields{
+			"instance-type": instanceType,
+		}).Warningf(
+			"Unable to find limits for instance type, consider setting --%s=true on the operator",
```

Comment:
Capitalize "operator"

Reply:
great, sorted! thanks for the very constructive review 🙇 !